ngraph/frontend/onnx_import/src/op/add.cpp

```diff
@@ -37,10 +39,15 @@ namespace ngraph
                         {
                             // Unidirectional broadcast right node to left shape.
                             auto axis = node.get_attribute_value<std::int64_t>("axis");
+                            auto axes_num = static_cast<size_t>(
+                                rhs_node.get_partial_shape().rank().get_length());
```

Comment:
Please check if rank is static, before you call `get_length()`

Reply:
How this function should handle dynamic rank?

Reply:
Anyway throwing meaningful message would be probably better than generic error from somewhere.